File: server/medicore.layer/MediCore.Service/Model/Custom/OperatingHourModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MediCore.Service.Model.Custom
{
    public class OperatingHourModel
    {
        //day integer https://msdn.microsoft.com/en-us/library/system.dayofweek.aspx
        public int Day { get; set; }
        public string StartTime { get; set; }
        public string EndTime { get; set; }
        public bool IsClossed { get; set; }
    }
}

File: server/medicore.apps/src/model/custom/operatingHourModel.ts
export class OperatingHourModel{
    //day interger https://msdn.microsoft.com/en-us/library/system.dayofweek.aspx
    day: number;
    startTime: string;
    endTime: string;
    isClossed: boolean;

    constructor();
    constructor(day?:number, startTime?:string, endTime?:string, isClossed?:boolean){
        this.day = day || 0;
        this.startTime = startTime || null;
        this.endTime = endTime || null;
        this.isClossed = isClossed || false;
    }
}